src/azul/queues.py

```diff
@@ -176,7 +176,6 @@ def wait_for_queue_level(cls, empty: bool = True, num_bundles: int = None):
         sleep_time = 5
         deque_size = 10 if empty else 1
         queues = cls.get_queues((config.notify_queue_name, config.document_queue_name))
```

Comment:
```suggestion
        queues = cls.get_queues(config.work_queue_names)
```
Now that `token_queue_name` has been removed from `work_queue_names` we can use it here.